src/App.tsx

```diff
@@ -38,24 +34,23 @@ export enum AlertLevel {
   NONE
 }
 
-interface L1ToL2MessageReaderWithNetwork extends IL1ToL2MessageReader {
+interface L1ToL2MessageReaderWithNetwork extends L1ToL2MessageReader {
   l2Network: L2Network;
 }
-
 const looksLikeCallToInboxethDeposit = async (
-  l1ToL2Message: IL1ToL2MessageReader
+  l1ToL2Message: L1ToL2MessageReader
 ): Promise<boolean> => {
-  const txData = await l1ToL2Message.getInputs();
-
+  const txData = await l1ToL2Message.messageData;
   return (
     txData.l2CallValue.isZero() &&
-    txData.maxGas.isZero() &&
-    txData.gasPriceBid.isZero() &&
-    txData.callDataLength.isZero() &&
-    txData.destinationAddress === txData.excessFeeRefundAddress &&
+    txData.gasLimit.isZero() &&
+    txData.maxFeePerGas.isZero() &&
+    (txData.data.toString() === ethers.constants.HashZero) &&
```

Comment:
I think this should be "" instead of HashZero (0x000000000...)

Reply:
I checked it with this tx Harry : 0xce3e6a4500f35b1d05838a2013af7f2a6bf8111ed0d9f223acc7339873d44d21
I'm not 100% sure that I'm doing it correct but I tried to do a legacy ETH deposit (tx hash above) for Nitro and then tested it with the panel.

Reply:
that particular tx seems to have `0x0000000000000000000000000000000000000000000000000000000000000000` as retryData, but I think normally one would use "" instead (less call data)

Reply:
Completely makes sense! Thanks!